examples/main.tf

```diff
@@ -8,6 +8,14 @@ resource "buildkite_team" "test" {
   default_member_role = "MEMBER"
 }
 
+resource "buildkite_team" "testtwo" {
```

Comment:
IS there any reason for this example resource instead of just using `test` for the `suite` too?

Reply:
More to simulate that a team not owning the suite (`team_owner_id` is added as either a RO/manage&read permissions) - we'd get the "already added to the suite" message if the owning team is attempted to be assigned to their own suite 👍